### ISSUE TITLE
Color contrast is too low in striped table blocks.

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#185]
**Describe the bug**
This is the only issue I could find when testing the current table block styles.

On dark backgrounds:
When a table block is added and the **striped** style is used, 
the text color is white over a light colored background and the color contrast is too low.
This happens both in the editor and the front.

**To Reproduce**
1. Go to the customizer and change the body background color to black
2. Create or open a post or page.
3. Add a table block with some content.
4. Select the striped block style
5. See that the text color is white and difficult to read.

The table block does not have a block sidebar option to change the text color, but can be changed inline.

**Screenshots**
![table-front](https://user-images.githubusercontent.com/7422055/94674474-81665d80-0318-11eb-8f0d-fb97dc8e5696.png)

**Comment by 2ndkauboy** [comment]
@carolinan what have you done to get a lighter background color for the second row? In my tests only the first one got a background color.

**Comment by carolinan** [comment]
> @carolinan what have you done to get a lighter background color for the second row? In my tests only the first one got a background color.

There is a background color option with a preset of colors in the block sidebar.

**Comment by 2ndkauboy** [comment]
Unfortunately it doesn't work yet for tables with a background set. Should I create a new issue for that? I'm already working on a patch.

**Comment by aristath** [comment]
Yes, a new issue would work best for this :+1:
